wg-ai-integration/charter.md

```diff
@@ -0,0 +1,105 @@
+on Kubernetes.
+
+The Working Group will provide a forum for a broad engineering community to
+give feedback to the project on challenges encountered when integrating with
+Kubernetes.
+
+This addresses a broad need with many end-users deploying complex AI systems,
+AI/ML platform providers, Kubernetes distributions, and developers of
+distributed AI applications facing these integration challenges. Standardizing
+solutions in this space benefits the entire Kubernetes ecosystem. Adjacent
+ecosystems could link to the outputs of this WG as a trusted vehicle for
+supporting AI integrations with Kubernetes.
+
+### In scope
+
+* Develop a shared community point of view and associated best practices
+enabling AI agent (or multi-agent) systems to integrate with Kubernetes.
+
+* Provide a forum for intersecting code experimentation in AI integration
+space (to be provided externally of Kubernetes via LF/CNCF) and discussion
+with the existing Kubernetes community.
+
+* Recommend an appropriate go forward governance model for AI Integrations
+with the Kubernetes project (existing SIGs, new SIG, or totally external
+to Kubernetes).
+
+* Identify appropriate auth(z) patterns for AI connector identities, its
+ closest caller, and Kubernetes RBAC.
+
+* Defining benchmarks on pros/cons of design approaches to meet user outcomes.
+
+* Ensure security, observability, and policy enforcement can be consistently
+applied across integrated systems (K8s and external Control Planes such as
+LLMs) and AI integration applications.
+
+* Define potential enhancements to API conventions to scale AI integration
+patterns that respect data privacy and safety concerns during our design
+process.
+
+* Reduce the complexity and custom development required for deploying,
+building and managing connectors of kubernetes API with AI agent ecosystems.
+
+### Out of Scope
+
+* Development of AI/ML frameworks or applications
+* General-purpose workload management not specific to AI/ML
+* Deploying inference workloads on Kubernetes (which is covered by WG Serving)
+* Manage accelerator devices (which is covered by WG Device Management)
+
+## Deliverables
+
+* The WG will provide space for collaboration and experimentation. If/when any
+ solid ideas emerge that require changes to Kubernetes (for example, updates
+ to kubectl for AI consumption), the WG will facilitate and coordinate the delivery
+ of KEPs and their implementations by the participating SIGs.
+* Interim artifacts will include documents capturing use cases, requirements,
+ integration architecture designs, and AI application communication patterns.
+* Establish best practices document for AI tool integration with Kubernetes and
+ a clear recommendation if/what set of reference tools may best fit in
+ Kubernetes project itself informed from data driven experimentation with
+ appropriate governance model.
+
+## Stakeholders
+
+* SIG Architecture
+* SIG API Machinery
+* SIG Apps
+* SIG Auth
+* SIG CLI
+* SIG Network
```

Comment:
The big areas where SIG Network has been hearing about / dealing with AI are DRA (aka "Manage accelerator devices", which is marked out-of-scope) and inference-related Gateway features (which would seem to fall under "Deploying inference workloads", which is also marked out-of-scope, and which is the subject of [a *different* AI WG proposal](https://groups.google.com/a/kubernetes.io/g/dev/c/XC_8qAyk8W0) anyway). None of the things you list as "In scope" above seem like they need input from SIG Network.

@kubernetes/sig-network-leads ?

Reply:
I defer to Dan, I really not very familiar with these protocols you mention but it seems to me both WG-Serving and the AI-GW proposal Dan is indicating will overlap on that area

Reply:
Thanks for the review. I added an item for this in the charter.

Reply:
So do we still need sig-network here or not? And where is the updated point, I can't find it :sweat_smile:

Reply:
I will ignore the first part of your response. I believe @shaneutt had a distinction betwen the current AI related working groups and the proposed. The question is fair, I am not contesting that, more asking for information on how to appropriately prioritize. We should have clear defined boundaries, however slight overlap isn't a bad thing at all. Shane is currently OOF, we should give him the benefit of the doubt this comment was missed? @shaneutt if you could address the concern from @danwinship? In the interest of transparency lets keep all conversations related to this PR here.